anymail/webhooks/postal.py

```diff
@@ -0,0 +1,169 @@
+import json
+from base64 import b64decode
+from datetime import datetime
+
+from django.utils.timezone import utc
+
+from .base import AnymailBaseWebhookView
+from ..exceptions import AnymailInvalidAddress, AnymailWebhookValidationFailure, AnymailImproperlyInstalled
+from ..utils import parse_single_address, get_anymail_setting
+from ..inbound import AnymailInboundMessage
+from ..signals import (
+    inbound,
+    tracking,
+    AnymailInboundEvent,
+    AnymailTrackingEvent,
+    EventType,
+)
+
+try:
+    from cryptography.exceptions import InvalidSignature
+    from cryptography.hazmat.backends import default_backend
+    from cryptography.hazmat.primitives import serialization, hashes
+    from cryptography.hazmat.primitives.asymmetric import padding
+except ImportError:
+    raise AnymailImproperlyInstalled(missing_package='cryptography', backend='postal')
```

Comment:
You'll need to use `_LazyError` here, or a missing cryptography will cause errors with other ESPs. See the amazon_ses webhooks for an example.